grafana/dashboard.json

```diff
@@ -73,15 +24,11 @@
   "editable": true,
   "fiscalYearStartMonth": 0,
   "graphTooltip": 0,
-  "id": null,
   "links": [],
   "liveNow": false,
   "panels": [
     {
-      "datasource": {
-        "type": "prometheus",
-        "uid": "${DS_PROMETHEUS}"
-      },
+      "datasource": {},
```

Comment:
This is not good, all panels should reference to the `${DS_PROMETHEUS}` variable or they won't work properly when you select another prometheus instance